integrations/anthropic/src/actions/generate-content.ts

```diff
@@ -86,14 +87,14 @@ export async function generateContent<M extends string>(
     if (err instanceof Anthropic.APIError) {
       const parsedInnerError = AnthropicInnerErrorSchema.safeParse(err.error)
       if (parsedInnerError.success) {
-        throw new UpstreamProviderError(
-          `Anthropic error ${err.status} (${parsedInnerError.data.error.type}) - ${parsedInnerError.data.error.message}`,
-          err
+        throw upstreamProviderFailedError(
```

Comment:
**nitpick:** I prefer error class `new UpstreamProviderFailedError` as it is more common

Reply:
I'd rather not do that because we would have a new custom error class again and also because extending the API error types isn't something we do currently, all of which may cause confusion as to the behavior of this error type.